[2021-01-05 23:11:29] ERROR_LOGS.ERROR: {"REQUEST":{"GET":[],"BODY":null,"REQUEST_METHOD":"GET","REQUEST_URI":"/users","REMOTE_ADDR":"1.236.152.18","HTTP_USER_AGENT":"PostmanRuntime/7.26.8"},"RESPONSE":{"code":500,"message":"SQL Exception -> #0 /var/www/dev/api-server/pdos/IndexPdo.php(11): PDOStatement->execute(Array)\n#1 /var/www/dev/api-server/controllers/IndexController.php(33): getUsers()\n#2 /var/www/dev/api-server/index.php(84): require('/var/www/dev/ap...')\n#3 {main}"}} [] []
